pkg/exporter/converters_test.go

```diff
@@ -56,6 +56,7 @@ func TestConversions(t *testing.T) {
 					},
 				},
 				Interfaces:    []model.IntfDir{model.NewIntfDir("eth0", model.DirectionEgress)},
+				UdnList:       []string{""},
```

Comment:
could you add some non-empty data in some of the tests?

Reply:
sure will do

Reply:
well actually this will require mock to ovnk decoded to return back strings